### ISSUE TITLE
Sycek doesn't parse C11 _Alignas() in structures correctly

### ISSUE THREAD

**Issue by le-jzr** [jxsvoboda/sycek#4]
An example of where this is useful:

```
typedef struct {
        _Alignas(16) uint8_t fpu[512];  /* FXSAVE & FXRSTOR storage area */
} fpu_context_t;

_Static_assert(_Alignof(fpu_context_t) >= 16, "fpu_context_t must be aligned");
```

Currently, sycek produces the following message:
```
Error: <kernel/arch/ia32/include/arch/fpu_context.h:41:18-19:num:16> unexpected, expected ')'.
```

**Comment by jxsvoboda** [comment]
Fixed in changeset e606f73fd875b5d9ff48261b23f69552a893f5d4. Note that use of_Alignas() with structure or union member is only defined in C17.

**Comment by le-jzr** [comment]
Awesome, thanks!
